examples/streamz/README.md

```diff
@@ -141,25 +132,50 @@ $ less cybert_workflow.log
     ```
     docker exec clx_streamz bash -c 'source activate rapids \
         && python $CLX_STREAMZ_HOME/python/<workflow_script> \
-        --broker <host:port> \
-        --input_topic <input_topic> \
-        --output_topic <output_topic> \
-        --group_id <kafka_consumer_group_id> \
+        --conf <configuration filepath> \
         --model <model filepath> \
         --label_map <labels filepath> \
         --poll_interval <poll_interval> \
         --max_batch_size <max_batch_size> \
         --benchmark <avg log size>'
     ```
     **Parameters:**
-    - `broker`* - Host and port where kafka broker is running. 
-    - `group_id`* - Kafka [group id](https://docs.confluent.io/current/installation/configuration/consumer-configs.html#group.id) that uniquely identifies the streamz data consumer.
-    - `input_topic` - The name for the input topic to consumer data.
-    - `output_topic` - The name for the output topic to send the output data.
+    - `conf` - The path to specify source and sink configuration properties.
```

Comment:
Do we need to update documentation in `docs/` with these changes? :)

Reply:
oh yeah, good catch. thnx!

Reply:
Updated docs. Thnx!